docs/Configuration.md

```diff
@@ -180,3 +180,50 @@ This is the key for Launch Darkly SDK access in our backend. There is one per en
 Read by `app-web`
 
 This is the client ID for Launch Darkly in our frontend. This key is designed to have limited access since it's exposed in our client side code. There is one per environment of local/dev/val/prod.
+
+## Email configuration
```

Comment:
Nice work on this email configuration section!

One suggestion. Could we link somewhere (confluence, figma?) that shows the exact emails for these stores. That way we have a reference to it. Maybe design has this made already somewhere.

Reply:
See new [Emails]( https://qmacbis.atlassian.net/wiki/spaces/OY2/pages/3164864517/Email) table of contents and[ the reference file](https://qmacbis.atlassian.net/wiki/spaces/OY2/pages/3162767369/Email+Configuration+Reference+for+PROD) on Confluence. If you have suggestions for those files welcomes comments on those files directly!

This process and seeing the current casing differences for what's on prod made me realize that we don't properly account for casing differences with emails in `pruneDuplicateEmails`. So also made a commit related to that [e941e73](https://github.com/CMSgov/managed-care-review/pull/1489/commits/e941e732803111e8c43d32031a2add1e49b27f3c).

Reply:
Nice catch!